src/components/Network.vue

```diff
@@ -179,7 +181,7 @@ class Network extends Vue {
         const client = await this.getNetworkClient();
         if (Network._hasOrSyncsOnTopOfConsensus) return client.headInfo.height;
         return new Promise((resolve) => this.$once(Network.Events.CONSENSUS_ESTABLISHED,
-            () => resolve(client.headInfo.height)));
+            () => setTimeout(() => resolve(client.headInfo.height), 10)));
```

Comment:
10 milliseconds is a bit arbitrary, don't you think? This is basically to wait a bit until the HEAD_CHANGED event fired after CONSENSUS_ESTABLISHED, yes? Do you think it makes sense to test for a hight > 1 every couple milliseconds and then resolve instead?

Reply:
Actually the `HEAD_CHANGED` event itself can be used which is being done now.

Reply:
Ha, thought so :D perfect!